crates/nargo/src/cli/prove_cmd.rs

```diff
@@ -108,21 +108,17 @@ pub fn prove_with_path<P: AsRef<Path>>(
     let proof =
         backend.prove_with_pk(compiled_program.circuit.clone(), solved_witness, proving_key);
 
-    println!("Proof successfully created");
     if check_proof {
         let valid_proof =
             verify_proof(compiled_program, public_inputs, return_value, &proof, verification_key)?;
-        println!("Proof verified : {valid_proof}");
+
         if !valid_proof {
             return Err(CliError::Generic("Could not verify generated proof".to_owned()));
```

Comment:
We could make use of the new error type here (although we'd have to deal with the case of a missing path.

Reply:
I've moved the check into verify_proof itself since all users of verify proof ended up wanting to return the same error.